akka-docs/src/test/scala/docs/persistence/query/PersistenceQueryDocSpec.scala

```diff
@@ -238,7 +238,8 @@ class PersistenceQueryDocSpec(s: String) extends AkkaSpec(s) {
     // assuming journal is able to work with numeric offsets we can:
 
     val blueThings: Source[EventEnvelope, NotUsed] =
-      readJournal.eventsByTag("blue")
+      readJournal.eventsByTag("blue",offset=Offset.NoOffset)
```

Comment:
I don't think you need the `offset=` prefix here - it should work without it, and doesn't really make the code much easier to read, right?

Reply:
But eventsByTag should have two parameter?
Or I missed someting?
trait EventsByTagQuery extends ReadJournal {

  def eventsByTag(tag: String, offset: Offset): Source[EventEnvelope, NotUsed]

}

Reply:
and looks like the formatting is wrong, please compile which will run the auto-formatter